src/shared/dissect-image.c

```diff
@@ -264,6 +265,47 @@ static int make_partition_devname(
         return asprintf(ret, "%s%s%i", whole_devname, need_p ? "p" : "", nr);
 }
 
+static int open_partition(const char *node, bool is_partition, LoopDevice *loop) {
```

Comment:
I think the signature should be:
```C
static int open_partition(const LoopDevice *loop, const char *node, bool is_partition)
```

Reply:
The main target is `node`. And `loop` is used for checking the opened file descriptor. I will add the `const` qualifier for `loop`.